wallet/udb/vsp.go

```diff
@@ -52,6 +94,24 @@ func GetVSPTicket(dbtx walletdb.ReadTx, tickethash chainhash.Hash) (*VSPTicket,
 	}
 	ticket := deserializeVSPTicket(serializedTicket)
 
+	host, err := GetVSPHost(dbtx, ticket.VSPHostID)
+	if err != nil && !errors.Is(err, errors.NotExist) {
+		// Only error out if it's not a 'not exist error'
+		return nil, err
+	}
```

Comment:
Same as above, you're ignoring any errors that are not `errors.NotExist`

Reply:
I believe this one is ok since it's !errors.Is(err, errorsNotExist).  So it should only be erroring on errors that aren't non exist errors.